diesel_derives2/src/util.rs

```diff
@@ -1,16 +1,20 @@
-use syn::*;
+use proc_macro2::Span;
 use quote::Tokens;
+use syn::*;
 
 pub use diagnostic_shim::*;
+use meta::*;
 
 pub fn wrap_in_dummy_mod(const_name: Ident, item: Tokens) -> Tokens {
+    let call_site = Span::call_site();
+    let use_everything = quote_spanned!(call_site=> __diesel_use_everything!());
```

Comment:
`call_site => __diesel_use_everything!()` ? It looks weird without the space

Reply:
I agree. Blame @dtolnay. https://docs.rs/quote/0.4.2/quote/macro.quote_spanned.html

> There should be no space before the => token.

Reply:
(But maybe we should just ignore that because I agree it looks really wrong without a space)

Reply:
Yeah the design constraints here are a bit unusual. If you have something like `quote_spanned!(a ??? b)` the objective was to *make* it look visibly off-balance and draw the eye a particular way, due to `a` being evaluated in the context of the procedural macro and `b` being evaluated in the generated code. I am happy with this design and I believe it has the intended effect.

Reply:
Thanks for the insight. <3